crates/typst/src/layout/transform.rs

```diff
@@ -335,6 +335,115 @@ cast! {
     length: Length => ScaleAmount::Length(length),
 }
 
+/// Skews content.
+///
+/// Skews an element in horizontal and/or vertical direction. The layout will
+/// act as if the element was not skewed unless you specify `{reflow: true}`.
+///
+/// # Example
+/// ```example
+/// #skew(-12deg)[This is some fake italic text.]
+/// ```
+#[elem(Show)]
+pub struct SkewElem {
+    /// The horizontal skewing angle.
+    ///
+    /// ```example
+    /// #skew(ax: 30deg)[Skewed]
+    /// ```
+    ///
+    #[parse(args.named("ax")?.or(args.find()?))]
+    #[default(Angle::zero())]
+    pub ax: Angle,
+
+    /// The vertical skewing angle.
+    ///
+    /// ```example
+    /// #skew(ay: 30deg)[Skewed]
+    /// ```
+    ///
+    #[parse(args.named("ay")?.or(args.find()?))]
```

Comment:
If we want to support a positional shorthand where both are the same (not sure whether it makes sense), we should do it like in `scale` and share the same result of `args.find()?`. Allowing just named or positional, but for both independently, is not great in my opinion.